app/views/users/show.html.erb

```diff
@@ -17,4 +17,4 @@
   <% end %>
 </div>
 
-<%= link_to "See all posts", "/users/#{@user.id}/posts?page=1" %>
+<%= link_to "See all posts", "/users/#{@user.id}/posts?page=1" %>
```

Comment:
- [ ] Dear @NalianAchei2020, Just as mentioned by the previouse reviewer, I face an issue when I click on `see all` to see all the posts. Please remember we need to route to the `index` action of the post to list all the posted for the user when we click this button. Please work on this.

### The issue 

Uploading Screen Recording 2023-11-04 at 1.14.55 in the afternoon.mov…